src/pages/registerPage/Register.tsx

```diff
@@ -1,8 +1,533 @@
+  align-items: center;
+  padding: 0px;
+  margin: 0;
+  width: 1400px;
+  height: 100%;
+`;
+
+const SideLeft = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0px;
+  width: 700px;
+  height: 100%;
+`;
+
+const ImgLeft = styled.img`
+  width: 700px;
+  height: 1200px;
+`;
+
+const SideRight = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0px;
+  width: 700px;
+  height: 100%;
+`;
+
+const ContainerRight = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 200px 24px 0px;
+  gap: 16px;
+  width: 600px;
+  height: 100%;
+`;
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  width: 280px;
+  height: 42px;
+  margin-top: -110px;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 16px 0px 0px;
+  gap: 16px;
+  width: 400px;
+  height: 934px;
+`;
+
+const InputComponent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 5px;
+  width: 400px;
+  height: 102px;
+  background: #ffffff;
+  & .inputBirthday {
+    width: 96%;
+    padding: 12.5px 8px;
+    color: rgba(0, 0, 0, 0.87);
+    font-size: 16px;
+    line-height: 23px;
+    border: 1px solid #c4c4c4;
+    border-radius: 4px;
+  }
+  & .helpText {
+    margin: 0 0 0 13px;
+    font-size: 12px;
+    position: relative;
+    color: grey;
+    top: 3px;
+    line-height: 20px;
+  }
+`;
+
+const DateBirthday = styled.div`
+width: 100%;
+}
+`;
+
+const DialogActions = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 12px 0px;
+  gap: 16px;
+  width: 420px;
+  height: 60px;
+  & .MuiButton-endIcon {
+    margin-right: 0;
+    margin-left: 0;
+    & .MuiSvgIcon-root {
+      width: 16px;
+    }
+  }
+`;
+
+const ButtonContinue = styled(Button)`
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  width: 110px;
+  height: 30px;
+  background: white;
+  box-shadow: unset;
+  font-weight: 500;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: #3f51b5;
+`;
+
+const Label = styled.label`
+  width: 400px;
+  height: 24px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: rgba(0, 0, 0, 0.87);
+  & > span {
+    color: red;
+  }
+`;
+
+type Inputs = {
+  cmnd: number;
+  email: string;
+  password: string;
+  name: string;
+  birthday: Date;
+  gender: string;
+  provinceId: number;
+  districtId: number;
+  wardId: number;
+};
+
+interface Province {
+  id: number;
+  name: string;
+}
+
+interface District {
+  id: number;
+  name: string;
+  provinceId: number;
+}
+
+interface Ward {
+  id: number;
+  name: string;
+  provinceId: number;
+  districtId: number;
+}
```

Comment:
```suggestion
interface Ward {
  id: number;
  name: string;
  districtId: number;
}
```

Dư thừa dữ liệu. Ward chỉ cần districtId